bin/getdatacenters.js

```diff
@@ -1,13 +1,19 @@
 'use strict';
 
+const fs = require('fs').promises;
 const PrxIpFilter = require('prx-ip-filter');
 
-const OUTFILE = `${__dirname}/../db/datacenters.json`;
+const OUTDIR = `${__dirname}/../db`;
+const OUTFILE = `${OUTDIR}/datacenters.json`;
 const BUCKET = 'prx-dovetail';
 const PREFIX = 'config/datacenters';
 
 // load datacenter csvs from S3, and save to db/datacenters.json
 async function run() {
+  try {
+    await fs.mkdir(OUTDIR);
+  } catch (err) {}
```

Comment:
Was trying to avoid the "directory already exists" error ... but re-reading, I think i can just pass the `{ recursive: true }` option and avoid the block entirely.